### PR TITLE
Hmusum/add logforwarding to schema

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

**Comment by hmusum** [comment]
@bjormel FYI or for review

File: config-model/src/main/resources/schema/admin.rnc [hunk 1]
```diff
@@ -109,3 +111,17 @@ ClusterControllers = element cluster-controllers {
...
 }
+
+LogForwarding = element logforwarding {
+    element forward {
+        attribute type {xsd:string } &
+        element source {
+            element log {  "access"| "vespa" }+
+        } &
+        element destination {
+            element endpoint { xsd:string } &
```
Comment (gjoranv) [inline config-model/src/main/resources/schema/admin.rnc]: Is this comma-separated? If so, I suggest instead to allow multiple 'endpoint' elements instead.

Comment (hmusum) [reply config-model/src/main/resources/schema/admin.rnc]: Yes, this was discussed at architect meeting and ended up like this, I'm afraid.

**Comment by bjormel** [comment]
We have gotten a request to support this in another way. Allowing the customer to configure the forwarder through a DeploymentServer.
E.g. something like shown below. I am testing this now in our splunk setup. We will then provide an  App for how the logs should be handled that the customer can modify. It is much more flexible, but might come with a risk.

cat deploymentclient.conf
[target-broker:deploymentServer]
targetUri = 10.222.140.222:5500

**Comment by bjormel** [comment]
@arnej27959

**Comment by arnej27959** [comment]
merged into another branch / PR

**Pull request closed without merging**